python/tvm/relay/frontend/tflite.py

```diff
@@ -1321,14 +1321,15 @@ def convert_gather(self, op):
         input_tensors = self.get_input_tensors(op)
         assert len(input_tensors) == 2, "input tensors length should be 2"
 
-        data = self.get_expr(input_tensors[0].tensor_idx)
-
+        if self.has_expr(input_tensors[0].tensor_idx):
+            data = self.get_expr(input_tensors[0].tensor_idx)
+        else:
+            data = self.exp_tab.new_const(self.get_tensor_value(input_tensors[0]),
+                                          dtype=self.get_tensor_type_str(input_tensors[0]\
+                                                                         .tensor.Type()))
```

Comment:
suggested to use `get_tensor_expr` for this code.

Reply:
Done